java-checks/src/main/java/org/sonar/java/checks/sustainability/AndroidExactAlarmCheck.java

```diff
@@ -49,15 +49,9 @@ protected MethodMatchers getMethodInvocationMatchers() {
   @Override
   protected void onMethodInvocationFound(MethodInvocationTree tree) {
     switch (ExpressionUtils.methodName(tree).name()) {
```

Comment:
I think I would prefer to go to something even simpler in our case:
```java
if (SET_WINDOW_NAME.equals(tree.methodSymbol().name())) {
  onSetWindowInvocation(tree);
} else {
  onSetExactInvocation(tree);
}
```
1. this prevents the uncovered line
2. it does not require `ExpressionUtils` and relies directly on semantic

Reply:
Yes, indeed. Fixed.